components/molecule/autosuggest/src/index.js

```diff
@@ -180,8 +179,22 @@ const MoleculeAutosuggest = ({
   }
 
   const handleClick = () => {
+    handleFocusInput()
```

Comment:
it could be changed to a direct assignation:
`const handleClick = handleFocusInput`
but i think its better leave it on this way just for not affecting to future click code iterations